src/widgetastic_patternfly/__init__.py

```diff
@@ -1804,3 +1804,48 @@ def items(self):
             # value will include the label from the <strong> block, parse it out
             items.update({key: element_text.replace(key, '', 1).lstrip()})
         return items
+
+
+class BreadCrumb(Widget):
+    """ Patternfly BreadCrumb navigation control
+
+    .. code-block:: python
+
+        breadcrumb = BreadCrumb()
+        breadcrumb.click_location(breadcrumb.locations[0])
+    """
+    ROOT = '//ol[@class="breadcrumb"]'
```

Comment:
May i suggest slightly different locator?
`//ol[contains(@class, "breadcrumb")]`?
Breadcrumb in satellite just have 2 classes, so we'll have to override it (which is still ok for us)

Reply:
Double checking MIQ breadcrumbs to know the scope, but sounds like a great improvement.